scripts/core/ui/components/ListPage/generic-list-page-item-view-edit.tsx

```diff
@@ -18,6 +18,7 @@ import {getFormFieldsFlat} from '../generic-form/get-form-fields-flat';
 import {hasValue} from '../generic-form/has-value';
 import {get, set} from 'lodash';
 import {produce} from 'immer';
+import {Button} from 'superdesk-ui-framework';
```

Comment:
Shouldn't this be from `'superdesk-ui-framework/react'`?

```suggestion
import {Button} from 'superdesk-ui-framework/react';
```